### PR TITLE
NETEXP-485

### PR DESCRIPTION
**Pull request by AkshayJagadish-ne**
Added functionality for overriding external host ports (in the AWLAN table) used by the AP

### CODE REVIEW

File: tip-wlan/charts/opensync-gw-cloud/values.yaml [hunk 1]
```diff
@@ -121,6 +121,13 @@ externalhostaddress:
   ovsdb: opensync-gw-cloud
```
Comment (rahulsharma2020) [inline tip-wlan/charts/opensync-gw-cloud/values.yaml]: I would group the externalhost address and ports together under `externalhost`.
So something like:
```
externalhost:
   address:
      ovsdb:
      mqtt:
   ports:
      ovsdb:
      mqtt:
```

File: tip-wlan/charts/opensync-gw-cloud/templates/deployment.yaml [hunk 2]
```diff
@@ -206,6 +206,10 @@ spec:
               value: {{ .Values.ethernetType.wanType }}
             - name: DEFAULT_WAN_NAME
               value: {{ .Values.ethernetType.wanName }}
+            - name: MQTT_BROKER_EXTERNAL_PORT
```
Comment (rahulsharma2020) [inline tip-wlan/charts/opensync-gw-cloud/templates/deployment.yaml]: If we can group the external variables together. Like `OVSDB_MANAGER`, `MQTT_SERVER_EXTERNAL` etc

File: tip-wlan/charts/opensync-gw-cloud/values.yaml (hunk 1 above)
Comment (AkshayJagadish-ne) [reply tip-wlan/charts/opensync-gw-cloud/values.yaml]: Thank you @rahulsharma2020, will make the change

**Review by rahulsharma2020 — APPROVED** [review]
Looks good now. Thanks @AkshayJagadish-ne

**Pull request merged**